.github/workflows/ci.yml

```diff
@@ -71,7 +71,5 @@ jobs:
       - name: Run tests
         run: bundle exec rake beaker
         env:
-          LANG: en_US
-          LC_ALL: en_US.UTF-8
```

Comment:
Wouldn't that break Puppet 7 runs on CentOS 7?

Reply:
I want to see if they fail. If that's the case I add it back